src/Data/Locales/pt_BR/messages.json

```diff
@@ -90,12 +90,12 @@
 	},
 	"settings@length__hint":
 	{
-		"message": "Comprimento recomendado: 8–16",
+		"message": "Comprimento recomendado: 8 - 16",
```

Comment:
No

Reply:
@XFox111 I need to leave it as before, because I thought it looked better this way, let me know and I'll correct it again!

Reply:
Yes, change it back

Reply:
Fix